sentences/es/cover_HassCloseCover.yaml

```diff
@@ -3,14 +3,14 @@ intents:
   HassCloseCover:
     data:
       - sentences:
-          - "cierra <name>"
-          - "cierra <name> <area>"
+          - "(cierra|cerrar|baja|bajar) <name>"
```

Comment:
Consider moving this to an expansion rule

Reply:
Done